### DISCUSSION TITLE
外部サービスの調査

### DISCUSSION THREAD

**Discussion by hayashi-ay** [katataku/minishell#2] [General]
折角なのでDXを向上させる外部サービスを導入したい。

気になるツール
- GitKraken
- Drovio
- Sider

**Comment by hayashi-ay** [comment]
[Drovio]

ペアプロ用のツール。無料でStandardプランが使用できる。

[Drovio]: https://app.drovio.com/account

**Reply by hayashi-ay** [reply]
Discord + VSCode(LiveShare)で行っていることができるイメージ。
追加で画面の録画や画面への描画もできそう。

今まで複数のアプリを立ち上げないとできなかったことが1つのアプリだけで完結できるのが良い？

**Reply by hayashi-ay** [reply]
VSCodeのLiveShareだとゲスト側の保存が機能しないことが多いので、これが改善されるなら導入もありかもしれない。

**Reply by katataku** [reply]
導入してデモ動画見ました！
https://help.drovio.com/en/article/what-is-drovio-md70g1/

**Reply by katataku** [reply]
実験しました。

マウスポインタが出てくるのが便利。
キーボード操作がうまくいかない？権限の問題かも

**Reply by katataku** [reply]
discord+VSCodeでいい説

**Reply by hayashi-ay** [reply]
全てが惜しいですね。discord + VSCode + Epic Pen or Miroで良さそう。

**Reply by hayashi-ay** [reply]
不採用。

**Comment by katataku** [comment]
[Git Kraken](https://www.gitkraken.com/github-student-developer-pack)

タスク管理ツール

**Reply by katataku** [reply]
https://www.youtube.com/watch?v=Y-kf3h2M_4c

GItHub Projectのかんばんボードと同期できるみたい。
オートメーションが強そう？

**Reply by hayashi-ay** [reply]
採用。

**Comment by hayashi-ay** [comment]
[Sider]

PRの分析ツール？

[Sider]:https://sider.review/github-student

**Reply by hayashi-ay** [reply]
レポジトリが片岡さんの方にあるので、片岡さん側で初期設定してもらう必要がありそう。

**Comment by hayashi-ay** [comment]
Github Organization使った方が便利ですかね？大げさな気もしますが。外部サービスとの連携周りはスムーズになりそう？

**Reply by hayashi-ay** [reply]
不採用。
